TAScheduler/models.py

```diff
@@ -23,16 +23,29 @@ class Section(models.Model):
     credits = models.IntegerField()
 
 
+# noinspection DuplicatedCode
 class TA(models.Model):
     user = models.ForeignKey(User, on_delete=models.CASCADE, null=True)
     sections = models.ManyToManyField(Section)
 
 
+# noinspection DuplicatedCode
+class TAToSection(models.Model):
```

Comment:
TATo  -> section IS NOT A JUNCTION TABLE - TA -> course NEEDS a junction table. Each section can only hold 1 TA, so it's impossible for it to be many - many